src/OpenTelemetry.Api/Trace/TelemetrySpanNew.cs

```diff
@@ -30,7 +30,11 @@ namespace OpenTelemetry.Trace
     public class TelemetrySpanNew : IDisposable
     {
         internal static readonly TelemetrySpanNew NoopInstance = new TelemetrySpanNew(null);
-        private readonly Activity activity;
+#pragma warning disable SA1304 // Other variable name won't fit here
+#pragma warning disable SA1307 // Other variable name won't fit here
+        internal readonly Activity activity;
```

Comment:
internal readonly Activity Activity;  - wont this work?

Reply:
This works. As the variable name is same of keyword name avoided using it. Changed it to Activity and removed all warnings.